audio/openal-soft/Portfile

```diff
@@ -52,6 +53,9 @@ patch.pre_args          -p1
 compiler.cxx_standard   2014
 compiler.thread_local_storage   yes
 
+compiler.blacklist      \
+                        {clang < 900}
```

Comment:
We can potentially make this less restrictive, but this seemed like a reasonable compromise. Let me know if you'd prefer to broaden the clang range we allow.